distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -3593,11 +3611,16 @@ TLS_RSA_WITH_AES_128_CBC_SHA
 Depending on the environment, it may be easier for integrators and administrators to configure {branding} using the Admin Console prior to disabling it for hardening purposes.
 The Admin Console can be re-enabled for additional configuration changes.
 ====
+
 In an environment hardened for security purposes, access to the {branding} Admin Console or the Felix Web Console might be denied.
-It is necessary to configure {branding} (e.g., providers, Schematron rulesets, etc.) using .config configuration files or the Admin Console. Configuration via the Karaf command line console is not supported and may result in configuration errors.
-The OSGi container detects the addition, updating, or deletion of `.config` files in the `etc/` directory.
+It is necessary to configure {branding} (e.g., providers, Schematron rulesets, etc.) using .config files or the Admin Console.
+Configuration via the Karaf command line console is not supported and may result in configuration errors.
+The OSGi container detects the creation of `.config` files in the `etc/` directory.
 The following sections describe how to configure each {branding} item using both of these mechanisms.
-A template file is provided for each configurable {branding} item so that it can be copied/renamed then modified with the appropriate settings.
+A template file is provided for some configurable {branding} items so that they can be copied/renamed then modified with the appropriate settings.
+
+//      DDF-1796: Hardening Section Needs a lot of Clean up
+//      Do we place the warning note here?
```

Comment:
I think I would just remove this comment. @ricklarsen, we may want to prioritize the tickets related to removing this section.